ros2cli_common_extension/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2cli_common_extensions</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="root@todo.todo">root</maintainer>
+  <license>TODO: License declaration</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <exec_depend>ros2action</exec_depend>
+  <exec_depend>ros2cli</exec_depend>
+  <exec_depend>ros2component</exec_depend>
+  <exec_depend>ros2doctor</exec_depend>
+  <exec_depend>ros2interface</exec_depend>
+  <exec_depend>ros2lifecycle</exec_depend>
+  <exec_depend>ros2lifecycle_test_fixtures</exec_depend>
```

Comment:
this one isn't needed

Reply:
Do you mean remove `ros2lifecycle_test_fixtures`?